webdriver-spec.html

```diff
@@ -4018,9 +4043,18 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
   for the <a>current top-level browsing context</a> for any reason,
   return <a>error</a> with <a>error code</a> <a>unsupported operation</a>.
 
- <li><p>Run the implementation-specific steps
-  to hide the <a>current top-level browsing context</a>
-  from the visible screen.
+ <li><p>Switching on the <a>visibility state</a>
+  of the <a>current top-level browsing context</a>’s <a>active document</a>:
+
+  <dl class=switch>
+   <dt><a data-lt="visibility normal">normal</a></dt>
+   <dt><a data-lt="visibility prerender">prerender</a></dt>
+   <dt><a data-lt="visibility unloaded">unloaded</a></dt>
+   <dd><p><a>Iconify the window</a>.
+
+   <dt><a data-lt="visibility hidden">hidden</a>
+   <dd><p><a>Restore the window</a>.
```

Comment:
Seems like this could be the main case, and the others handled with an `otherwise` case. This would handle the case of a vendor-prefixed DOMString, which is allowed as a valid visibility state.